spec/unit/knife/core/windows_bootstrap_context_spec.rb

```diff
@@ -148,22 +148,26 @@
 
   describe "#config_content" do
     before do
+      root_path = windows? ? "c:/chef" : "/chef"
+
       bootstrap_context.instance_variable_set(:@chef_config, Mash.new(config_log_level: :info,
         config_log_location: STDOUT,
         chef_server_url: "http://chef.example.com:4444",
         validation_client_name: "chef-validator-testing",
-        file_cache_path: "c:/chef/cache",
-        file_backup_path: "c:/chef/backup",
-        cache_options: ({ path: "c:/chef/cache/checksums", skip_expires: true })))
+        file_cache_path: windows? ? "#{root_path}/cache" : "#{root_path}/cache",
+        file_backup_path: windows? ? "#{root_path}/backup" : "#{root_path}/backup",
```

Comment:
These don't need a condition in them.
```suggestion
        file_cache_path: "#{root_path}/cache",
        file_backup_path: "#{root_path}/backup",
```